src/commonControls/dateRangeControl/DateRangeControl.ts

```diff
@@ -163,6 +163,14 @@ export interface DateRangeControlProps extends ContainerControlProps {
      * Props to configure input handling.
      */
     inputHandling?: ControlInputHandlingProps;
+
+    /**
+     * Function that maps the date(s) / date range slotValue to a corresponding
+     * rendered value that will be presented to the user.
+     *
+     * Default: returns the slotValue unchanged.
+     */
+    valueRenderer?: (slotValue: string, input: ControlInput) => string;
```

Comment:
'slotValue' doesn't seem right.  for this, and perhaps others, I'd suggest just making it 'value' and passing `control.state.value`.  I notice that DateRangeControl isn't actually following the convention -- it should be putting both dates into an object called value.  Currently there is a .value (by inheritance from ControlState) that is never used.

Because dateRange sometimes needs to just render a single date, it seems to require two renderer props.

```
valueRenderer: ( value: DateRangeValue, ..) =>
dateRenderer: (date: string, ...) =>
```
We don't have a convention yet on the name for the first one.. it could reasonably be 'dateRangeRenderer' but maybe its good to have a convention that render the whole value is called 'valueRenderer'

Reply:
I can see that giving a nice name to the parameter could be good here and elsewhere.  So I'd suggest one of the following:
```
valueRenderer: (dateRange: DateRangeValue) =>
or
valueRenderer: (startDate: string, endDate:s string) =>
```

final thought:  because there are multiple render function, and some controls might need lots of these, perhaps we should group them.  This would also make it much cleaner to offer different render functions for startDate vs endDate.

```
valueRenderer: {
  startDate: 
  endDate:
  range:
}

```

Reply:
Updated func signature to `valueRenderer?: (value: DateRange, component: string, input: ControlInput) => string;` where component = ['START_DATE'|'END_DATE'|'RANGE'] to resolve the render functional behaviour based on individual definitions/customizations done by developer if required.

Reply:
nice!